gameDetails.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+require 'vendor/autoload.php';
+use GameListing\Hydrators\GamesHydrator;
+
+$db = new PDO('mysql:host=db;dbname=games', 'root', 'password');
+$gameId = $_GET['id'];
```

Comment:
need to validate this in some way or handle what happens if the id doesn't exist - currently it does this:
![image](https://user-images.githubusercontent.com/25844355/133642150-0620ddfb-56f6-4404-8de7-1bc3e6447ae9.png)

Reply:
fixed